08_project/mohankrishna/pkg/mohankrishna-puppy/store/leveldb_store.go

```diff
@@ -0,0 +1,79 @@
+package store
+
+import (
+	"encoding/json"
+	"fmt"
+
+	types "github.com/anz-bank/go-training/08_project/mohankrishna/pkg/mohankrishna-puppy"
+	"github.com/syndtr/goleveldb/leveldb"
+)
+
+type LevelDBStore struct {
+	ldb *leveldb.DB
+}
+
+func NewLevelDBStore(dbPath string) *LevelDBStore {
+	db, _ := leveldb.OpenFile(dbPath, nil)
+	return &LevelDBStore{db}
+}
+
+func (ls *LevelDBStore) CloseDB() {
+	ls.ldb.Close()
+}
+
+func (ls *LevelDBStore) CreatePuppy(puppy *types.Puppy) error {
```

Comment:
Similar to my comments in sync store, this storer currently has race conditions. I'll leave it up to you to decide if you want this store to be thread-safe.